tests/workflows/basic_tool_calling_node/tests/test_workflow.py

```diff
@@ -542,6 +542,52 @@ def generate_prompt_events(*_args, **_kwargs) -> Iterator[ExecutePromptEvent]:
     # THEN the adhoc_execute_prompt_stream should be called exactly twice
     assert vellum_adhoc_prompt_client.adhoc_execute_prompt_stream.call_count == 2
 
+    # AND the second call should have the correct blocks structure
+    second_call = vellum_adhoc_prompt_client.adhoc_execute_prompt_stream.call_args_list[1]
+    assert second_call.kwargs["blocks"] == [
```

Comment:
sorry I made a mistake. We actually just want to assert the value of the `chat_history` input in this kwarg on this invocations. Please iterate